Cargo.toml

```diff
@@ -36,24 +36,22 @@ yaml = ["yaml_config"]
 
 proxy = ["tokio-socks"]
 
-tls-native = ["native-tls", "tokio-tls"]
+tls-native = ["native-tls", "tokio-native-tls"]
 tls-rust = ["tokio-rustls", "webpki-roots"]
 
 
 [dependencies]
 bufstream = "0.1.0"
-bytes = "0.5.0"
+bytes = "0.6.0"
```

Comment:
`bufstream` and `bytes` dependencies are not used by the crate.
I guess we can directly removed them.